aptos-move/framework/move-stdlib/sources/configs/features.move

```diff
@@ -85,14 +85,15 @@ module std::features {
         is_enabled(VM_BINARY_FORMAT_V6)
     }
 
-    /// Whether gas fees are collected and distributed to the block proposers.
-    /// Lifetime: transient
-    const COLLECT_AND_DISTRIBUTE_GAS_FEES: u64 = 6;
-
-    public fun get_collect_and_distribute_gas_fees_feature(): u64 { COLLECT_AND_DISTRIBUTE_GAS_FEES }
+    #[deprecated]
+    /// Deprecated feature
+    public fun get_collect_and_distribute_gas_fees_feature(): u64 {
```

Comment:
I am not sure abort here is a good practice since anyone can call this func even it is not used by us.